pkgs/applications/system/qjournalctl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ 
+  stdenv,
+  fetchFromGitHub,
+  qtbase,
+  qmake,
+  pkgconfig,
+  libssh,
+	wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qjournalctl";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "pentix";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "08qfcf5yrfzsis9gkawkhshwmmi0jqw8xv23925hqass4a2xz704";
+  };
+
+  postPatch = ''
+    substituteInPlace qjournalctl.pro --replace /usr/ $out/
```

Comment:
It might be nice to have this configurable through `makeFlags` like here: https://github.com/corrados/jamulus/pull/247

Reply:
I'm trying to pick up this PR. How could this be achieved? By providing a patch for the .pro file?